File: phy40/compare.c
// compare.c - - Input two number, compare them, and print the sum
#include <stdio.h>

int main() {
    int a, b, sum;

    // Prompt and read a
    printf("Enter value for a:\n");
    scanf("%d", &a);

    // Prompt and read b
    printf("Enter value for b:\n");
    scanf("%d", &b);

    sum = a + b;

    // Conditional check
    if (a > b) {
        printf("a is greater than b\n");
    } else {
        printf("a is less than or equal to b\n");
    }

    // Print formatted output
    printf("a=%d, b=%d, a+b=%d\n", a, b, sum);

    return 0;
}

File: phy40/compare.f
C compare.f - Input two number, compare them, and print the sum
      PROGRAM COMPARE_INPUT
C     Reads two integers, compares them, and prints the results

      INTEGER A, B, SUM

C     Prompt and read A
      PRINT *, 'Enter value for a:'
      READ *, A

C     Prompt and read B
      PRINT *, 'Enter value for b:'
      READ *, B

      SUM = A + B

C     Conditional check
      IF (A .GT. B) THEN
         PRINT *, 'a is greater than b'
      ELSE
         PRINT *, 'a is less than or equal to b'
      ENDIF

C     Print formatted values
      PRINT *, 'a =', A, ', b =', B, ', a+b =', SUM

      END
      
      
